### PR TITLE
Added Heron's

### PR DESCRIPTION
**Pull request by binoy-d**
filled out the code for heron's formula

### CODE REVIEW

File: Geometry/Heron.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		System.out.print("What is the length of side 3?");
+		double c = scan.nextDouble();
+
+		System.out.println("Calculating...");
+
+    //Calculate the area
+		double s = (double)(0.5*(a+b+c));
+		double answer = (double)Math.sqrt(s*(s-a)*(s-b)*(s-c));
+
+    //Print out zeroes calculated from the Heron's formula
```
Comment (Kyle-P) [inline Geometry/Heron.java]: Comment should say "Print out area calculated from the Heron's formula" instead of "zeroes."

**Comment by binoy-d** [comment]
fixed it

File: Geometry/Heron.java (hunk 1 above)
Comment (binoy-d) [reply Geometry/Heron.java]: takrn care of

**Comment by Kyle-P** [comment]
Thanks @binoy-d!

**Pull request merged**